blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/OrganizationImpl.java

```diff
@@ -128,4 +143,38 @@ private Object wrap(Object action){
     private boolean isExportedBean(Class clz){
         return clz.getAnnotation(ExportedBean.class) != null;
     }
+
+    private static final Pattern pattern = Pattern.compile("/blue/organizations/([^/]*)/");
+
+    /**
+     * Gets the organization from the URL in the form of:
+     * 
+     * <pre>
+     * /blue/organizations/ORG_NAME/...
+     * </pre>
+     * 
+     * Defaulting to the first organization if not found
+     * 
+     * NOTE: designed for exclusive use in preloaders or places where its not possible to get an organization ref in
+     * scope
+     * 
+     * @return the organization name if found, {code}null{code} if not
+     */
+    @CheckForNull
+    @Restricted(None.class) //Should only be used for preloaders
+    public static BlueOrganization getOrganizationFromURL() {
```

Comment:
We should get BlueOrganization thru constructor, should be passed around from caller. Should follow the same pattern as on master. 🐛